es74x-cloud/src/main/scala/tech/beshu/ror/proxy/es/providers/ProxyIndexJsonContentManager.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ *     Beshu Limited all rights reserved
+ */
+package tech.beshu.ror.proxy.es.providers
+
+import java.util
+
+import monix.eval.Task
+import tech.beshu.ror.es.IndexJsonContentManager
+
+// todo: implement
+object ProxyIndexJsonContentManager extends IndexJsonContentManager {
+
+  override def sourceOf(index: String,
+                        `type`: String,
+                        id: String): Task[Either[IndexJsonContentManager.ReadError, util.Map[String, _]]] =
+    Task.now(Left(IndexJsonContentManager.CannotReachContentSource))
+
+  override def saveContent(index: String,
+                           `type`: String,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [`type` is a reserved keyword don't use it as a name](https://app.codacy.com/app/scarduzio/elasticsearch-readonlyrest-plugin/pullRequest?prid=4591272)